system/Config/DotEnv.php

```diff
@@ -140,7 +140,7 @@ public function normaliseVariable(string $name, string $value = ''): array
      */
     protected function sanitizeValue(string $value): string
     {
-        if (! $value) {
+        if ($value === '' || $value === '0') {
```

Comment:
This is not a bug, but no need to treat only `'0'`.

Reply:
do you mean only check empty string `''` ?

Reply:
Yes, remove `|| $value === '0'`.